docs/index.rst

```diff
@@ -19,6 +19,7 @@ Getting Started
    1-concepts
    tutorials
    api/scarlet
+   regression
```

Comment:
nice that this is included publicly. I'm not sure anyone would use it besides us, but it speaks to our desire for transparency.